cmd/yggd/worker.go

```diff
@@ -192,6 +192,18 @@ func stopWorker(name string) error {
 		return fmt.Errorf("cannot parse data: %w", err)
 	}
 
+	procCmdline, err := os.ReadFile(fmt.Sprintf("/proc/%v/cmdline", pid))
+	if err != nil {
+		return fmt.Errorf("cannot read file /proc/%v/cmdline: %v", pid, err)
```

Comment:
@Archana-PandeyM tested that returning error is not the good idea. Why? When `workers/worker_name.pid` exists and it contains PID of non-existent process, then rhcd will not be able to start. We should only print some warning and delete `workers/worker_name.pid` file.

Reply:
Hmm. I wrote this error check under the assumption of other failures related to
reading the proc file (permissions, etc.) but you're right. If /proc/$PID does
not exist, the process is already stopped. Maybe we should check for the
existence of `/proc/$PID` explicitly, first?